Engine/platform/base/agsplatformdriver.cpp

```diff
@@ -86,6 +87,27 @@ void AGSPlatformDriver::GetSystemTime(ScriptDateTime *sdt) {
     sdt->year = newtime->tm_year + 1900;
 }
 
+void AGSPlatformDriver::DisplayAlert(const char *text, ...)
+{
+    char displbuf[1024];
```

Comment:
I am probably paranoid, but it seems we use 2000 and now this is 1024. I would keep it at 2000 just to be safe.